### PR TITLE
feat: Add FAQ Section

### PR DESCRIPTION
**Pull request by padmajabhol**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
This PR adds a FAQ section.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by riteshsp2000** [comment]
Hey @padmajabhol. A few changes before I go ahead and review the PR

1. The section is not responsive. As in before something like md breakpoint, the section should occupy the entire width of the screen.
2. Ask @CIPHERTron to upload the images that you have used to cloudinary and use those links instead of the images.
3. Try to break your work into multiple commits, pushing all your work in one commit might be problematic down the line. As in consider some part of the code that needs to be reverted in the PR, having just one commit will avoid reverting that specific part. Also branch name, from next PR onwards name it as per the work (in this case something like faq-section) 

The preview looks all good, ill go throw the code and if any changes, ill suggest those latest by tomorrow

File: src/components/FaqSection/Faq/Faq.styles.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
```
Comment (riteshsp2000) [inline src/components/FaqSection/Faq/Faq.styles.js:1]: Rename this file to .jsx . JSX files are files which export a component, JS files are just normal javascript files

File: src/components/FaqSection/Faq/Faq.styles.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        justify-center
+        bg-purple-light
+        rounded-full
+        relative
+        text-black
+        border-2 border-gray-800
+    `}
+`
+
+const Bodydark = styled.p`
```
Comment (riteshsp2000) [inline src/components/FaqSection/Faq/Faq.styles.js:56]: I am not sure if we have created this typography, but we surely have a body typography in shared folder. Check once.

File: src/components/FaqSection/Faq/FaqQA.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const STAGE = {
```
Comment (riteshsp2000) [inline src/components/FaqSection/Faq/FaqQA.jsx:1]: Rename this file to .js instead of jsx

**Review by riteshsp2000 — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]

**Comment by CIPHERTron** [comment]
@all-contributors add @padmajabhol for Code

**Pull request merged**